src/main/java/org/openstreetmap/atlas/geography/atlas/command/OsmPbfToAtlasSubCommand.java

```diff
@@ -0,0 +1,207 @@
+
+import java.io.PrintStream;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.apache.commons.io.FilenameUtils;
+import org.openstreetmap.atlas.geography.MultiPolygon;
+import org.openstreetmap.atlas.geography.atlas.pbf.AtlasLoadingOption;
+import org.openstreetmap.atlas.geography.atlas.pbf.OsmPbfLoader;
+import org.openstreetmap.atlas.geography.boundary.CountryBoundaryMap;
+import org.openstreetmap.atlas.streaming.resource.File;
+import org.openstreetmap.atlas.tags.filters.ConfiguredTaggableFilter;
+import org.openstreetmap.atlas.utilities.configuration.StandardConfiguration;
+import org.openstreetmap.atlas.utilities.runtime.Command.Optionality;
+import org.openstreetmap.atlas.utilities.runtime.Command.Switch;
+import org.openstreetmap.atlas.utilities.runtime.Command.SwitchList;
+import org.openstreetmap.atlas.utilities.runtime.CommandMap;
+import org.openstreetmap.atlas.utilities.runtime.FlexibleSubCommand;
+
+/**
+ * This command converts an OSM PBF file to an Atlas file. It requires the path to a pbf and an
+ * output file. It also takes a number of optional parameters.
+ *
+ * @author bbreithaupt
+ */
+public class OsmPbfToAtlasSubCommand implements FlexibleSubCommand
+{
+    private static final String NAME = "pbf-to-atlas";
+    private static final String DESCRIPTION = "Converts a PBF to an Atlas file.";
+
+    // Required parameters
+    private static final Switch<File> INPUT_PARAMETER = new Switch<>("pbf", "Input PBF path",
+            File::new, Optionality.REQUIRED);
+    private static final Switch<File> OUTPUT_PARAMETER = new Switch<>("output",
+            "Output Atlas file path", File::new, Optionality.REQUIRED);
+
+    // Filter parameters
+    private static final Switch<File> EDGE_FILTER_PARAMETER = new Switch<>("edge-filter",
+            "Path to a json filter for determining Edges", File::new, Optionality.OPTIONAL);
+    private static final Switch<File> NODE_FILTER_PARAMETER = new Switch<>("node-filter",
+            "Path to a json filter for OSM nodes", File::new, Optionality.OPTIONAL);
+    private static final Switch<File> RELATION_FILTER_PARAMETER = new Switch<>("relation-filter",
+            "Path to a json filter for OSM relations", File::new, Optionality.OPTIONAL);
+    private static final Switch<File> WAY_FILTER_PARAMETER = new Switch<>("way-filter",
+            "Path to a json filter for OSM ways", File::new, Optionality.OPTIONAL);
+    private static final Switch<File> WAY_SECTION_FILTER_PARAMETER = new Switch<>(
+            "way-section-filter", "Path to a json filter for determining where to way section",
+            File::new, Optionality.OPTIONAL);
+
+    // Load Parameters
+    private static final Switch<Boolean> LOAD_RELATIONS_PARAMETER = new Switch<>("load-relations",
+            "Whether to load Relations (boolean)", Boolean::new, Optionality.OPTIONAL);
```

Comment:
How about `Boolean::parseBoolean` and add a default value of `"true"` at the end? That would make the switches well defined for the end user!